internal/nodes/avi_model_l4_translator.go

```diff
@@ -662,3 +650,23 @@ func buildPoolWithL4Rule(key string, pool *AviPoolNode, l4Rule *akov1alpha2.L4Ru
 
 	utils.AviLog.Debugf("key: %s, msg: Applied L4Rule %s configuration over Pool %s", key, l4Rule.Name, pool.Name)
 }
+
+// In case the VS has services that are a mix of TCP and UDP/SCTP sockets,
+// we create the VS with global network profile TCP Proxy or Fast Path based on license,
+// and override required services with UDP Fast Path or SCTP proxy. Having a separate
+// internally used network profile (MIXED_NET_PROFILE) helps ensure PUT calls
+// on existing VSes.
+func getNetworkProfile(isSCTP, isTCP, isUDP bool) string {
+	if isSCTP && !isTCP && !isUDP {
+		return utils.SYSTEM_SCTP_PROXY
+	} else if isTCP && !isUDP && !isSCTP {
```

Comment:
@abhishekbsingh As discussed, can you please check this scenario for advanced l4. The network profile was previously set to TCP fast path here but now being set based on license.

Reply:
I think it should be okay, nit: else is not required as each if block has a return in it

Reply:
Have removed the else statements.